src/components/layout/header/header.module.scss

```diff
@@ -24,7 +24,9 @@
 
 .get-involved {
   display: inline-block;
-  line-height: 1.75;
+  @media (min-width: $viewport-lg) {
+    line-height: 1.75;
+  }
   padding: 0.4rem 0.5rem; // ignore-style-rule
   transition: all 0.2s;
   height: 35px; // match the search box height
```

Comment:
@schwartzadev I was thinking we could move this explicit height to the viewport-lg and above media query above, since we only need to match the search box height at that range? I think this will help the vertical centering at the smaller breakpoints as well.